app/assets/stylesheets/replies.scss

```diff
@@ -73,7 +74,6 @@
     position: relative;
     width: 130px;
     word-wrap: break-word;
-    display: inline-block;
```

Comment:
Are we sure that this can be removed without changing anything?

Reply:
Yes because -- if you expand it's also there five lines up.

Reply:
Oh, cool! I didn't think it'd be duplicated.